### DISCUSSION TITLE
Possible mistake in introduction, explaining LCS definition

### DISCUSSION THREAD

**Discussion by Gerstbreu** [StructuralAnalysisFormat/StructuralAnalysisFormat-Doc#77] [Q&A] [answered]
In the introduction (https://www.saf.guide/en/stable/getting-started/introduction.html) there is the sentence "In case, this is not possible (wall in plane/or parallel to global Y global X plane), then local Z follows the positive half space defined by positive global Y and plane defined by global X and global Z axis.". 

Is that right? I think "parallel to global X global Z plane" would make sense.

**Comment by sonkaste17** [comment, marked as answer]
Hi,
you are right. Thank you for noticing. We are going to correct it. 
There should be " ... parallel to global X global Z plane ... "

![image](https://user-images.githubusercontent.com/81751485/149145828-1dae6b36-0a0b-4810-8df6-49689b5409b4.png)
